dpnp/backend/kernels/dpnp_krnl_indexing.cpp

```diff
@@ -175,6 +175,25 @@ void dpnp_fill_diagonal_c(void* array1_in, void* val_in, size_t* shape, const si
     return;
 }
 
+template<typename _DataType>
+void dpnp_place_c(void* arr_in, void* mask_in, void* vals_in, const size_t arr_size, const size_t vals_size)
```

Comment:
```suggestion
void dpnp_place_c(void* arr_in, long* mask_in, void* vals_in, const size_t arr_size, const size_t vals_size)
```

Reply:
or maybe it is better use (alloc) char array the size of given array for ones and zeros.